.circleci/run_simulation_sweeps.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import os
+import subprocess
+import six
+import shutil
+import tempfile
+
+models= ["minerva"]
+
+def run_sim_sweeps(sweep_file, gem5_binary, num_threads):
+  sweeps_dir = "%s/experiments/sweeps" % os.environ["SMAUG_HOME"]
+  sweep_generator = os.path.join(sweeps_dir, "main.py")
+
+  for model in models:
+    temp_dir = tempfile.mkdtemp(dir=os.getcwd())
+    process = subprocess.Popen([
+        "python", sweep_generator, "--model", model, "--params", sweep_file,
```

Comment:
I was confused where this file went, then I realized it's in the experiments submodule. We should move as much of the code from that into here, so experiments is literally just a dumb directory full of model files, parameters, and simulation configuration files. I have not carefully looked at even a single line of code in that main.py file yet.

Reply:
Agreed. I will move the sweep generator to here.